internal/character/march7th/technique.go

```diff
@@ -0,0 +1,30 @@
+package march7th
+
+import (
+	"github.com/simimpact/srsim/internal/global/common"
+	"github.com/simimpact/srsim/pkg/engine/info"
+	"github.com/simimpact/srsim/pkg/key"
+	"github.com/simimpact/srsim/pkg/model"
+)
+
+func (c *char) Technique(target key.TargetID, state info.ActionState) {
+	freezeTarget := c.engine.Retarget(info.Retarget{
+		Targets: c.engine.Enemies(),
+		Filter: func(target key.TargetID) bool {
+			return c.engine.Stats(target).GetDebuffRES(model.BehaviorFlag_STAT_CTRL_FROZEN) <= 0.8 &&
+				c.engine.Stats(target).GetDebuffRES(model.BehaviorFlag_STAT_CTRL) <= 0.8
+		},
```

Comment:
`GetDebuffRES` takes in multiple flags by design (varags param) for cases such as this where there are multiple being searched. In this case, only the highest value is taken. We want to avoid calling `Stats` multiple time since it is an expensive operation.

```go
Filter: func(target key.TargetID) bool {
	return c.engine.Stats(target).GetDebuffRES(
      model.BehaviorFlag_STAT_CTRL_FROZEN, model.BehaviorFlag_STAT_CTRL) <= 0.8
},
```